src/db.py

```diff
@@ -263,3 +263,21 @@ def get_contributors(self, username, submission_name):
         if submission_doc is None:
             raise SubmissionDoesNotExistError(str(username) + ": " + str(submission_name))
         return submission_doc["contributors"]
+
+
+    def get_projects(self, username):
+        project_db = self.mongo.gitsubmit.projects
+        resultCursor = project_db.find({"owner": username})
+        return resultCursor.toArray()
+
+
+    def get_submissions(self, username):
+        submission_db = self.mongo.gitsubmit.submissions
+        resultCursor = submission_db.find( { "$or": [ {"owner": username}, {"contributors": username} ] })
```

Comment:
Did you test out this query? I think contributors needs to be an `"$in"` query, but I remember mongo sometimes doing things correctly automatically